skietbaan_fe/src/components/RegisterMemberStyles.css

```diff
@@ -1,26 +1,22 @@
 @media all and (min-width: 480px) {
-    .Register {
```

Comment:
class names in css are standard to lower case

Reply:
making the changes